manifests/addons/gen.sh

```diff
@@ -26,30 +26,17 @@ ADDONS="${WD}/../../samples/addons"
 DASHBOARDS="${WD}/dashboards"
 mkdir -p "${ADDONS}"
 
-# Set up Kiali. The upstream only has an operator, so we will deploy that
-# We set up anonymous authentication as this is for demos
-KIALI_SRC=$(mktemp -d)
-KIALI_VERSION=1.18
-pushd "${KIALI_SRC}"
-curl -s -L https://github.com/kiali/kiali-operator/archive/v${KIALI_VERSION}.tar.gz | tar xz
-./kiali-operator-${KIALI_VERSION}/deploy/merge-operator-yaml.sh \
-  -f "${ADDONS}/kiali.yaml" --operator-image-version v${KIALI_VERSION} --operator-namespace istio-system --create-operator-namespace false --cluster-role-creator true
-cat <<EOF >> "${ADDONS}/kiali.yaml"
----
-apiVersion: kiali.io/v1alpha1
-kind: Kiali
-metadata:
-  name: kiali
-  namespace: istio-system
-spec:
-  deployment:
-    accessible_namespaces: ["**"]
-  auth:
-    strategy: anonymous
-EOF
-popd
-# Remove timestamp from kiali so its stable
-sed -i 's/# Auto-generated by merge-operator-yaml.sh.*//' "${ADDONS}/kiali.yaml"
+# Set up kiali
```

Comment:
We are checking in the result of gen.sh, so I think youll actually need to run this. That way users just run `kubectl apply`

Reply:
> I think youll actually need to run this. 

Not sure what you mean. Run what?

If you mean that "sed" line, that "# Auto-generated" line came from the old "merge" script that we aren't using anymore. So that's moot. We would be using helm just like prometheus and grafana. I tried to follow the same paradigm used when generating the prom and grafana templates.

As for:

> That way users just run `kubectl apply`

That's not going to work - not with addons that require CRDs. See: https://github.com/istio/istio/issues/25889

Reply:
I mean run `manifests/addons/gen.sh` and check in the result. Users are not running this script, its just to generate the plain text yamls that users use

Reply:
Oh... you mean PR authors should run "gen.sh" and check in the results that output? Now I get it. OK, I'll do that.

Reply:
Yep, I didn't add it to the `gen` script since its actually pulling from live sources so I wasn't 100% sure it would be deterministic.

Reply:
OK, I just pushed a new commit with the yaml.